mungegithub/mungers/approvers/owners.go

```diff
@@ -396,40 +396,57 @@ func (ap Approvers) IsApproved() bool {
 	return ap.UnapprovedFiles().Len() == 0
 }
 
-// ListString returns the list of all approvers along with links and tooltips.
-func (ap Approvers) ListString() string {
-	approvals := []string{}
+// ListApprovals returns the list of approvals
+func (ap Approvers) ListApprovals() []Approval {
+	approvals := []Approval{}
 
 	for _, approver := range ap.GetCurrentApproversSet().List() {
-		approvals = append(approvals, ap.approvers[approver].String())
+		approvals = append(approvals, ap.approvers[approver])
 	}
 
-	return strings.Join(approvals, ", ")
+	return approvals
 }
 
 type File interface {
-	toString(string, string) string
+	String() string
 }
 
 type ApprovedFile struct {
```

Comment:
I'm using `String() string` that is the idiomatic way in go to stringify an object. This makes it even more easy/obvious to call from the template.